hail-ci-deploy.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
```

Comment:
I don't see a `hail-ci-deploy.sh` for `upload`?

Also, we should probably have a `hail-ci-build.sh` that at least starts the server once to check the python syntax.

Reply:
I guess there's the issue of the secrets. Maybe create some bogus secrets? `upload` does not appear to eagerly use the secrets.

Reply:
> I guess there's the issue of the secrets.

What's the issue?  I don't understand.

Reply:
I'm pushing back on the fact that this has no tests at all, not even a test that it has correct python syntax. The issue with the secrets is that `upload` requires some secret information to run, but `hail-ci-build.sh` is executed in an environment that does not have those secrets.

Reply:
Can I follow up with another PR that runs the script? I need to rebuild the image with a new environment for upload.